### PR TITLE
Map loader

### PR DESCRIPTION
**Pull request by n2oneProgrammer**

### CODE REVIEW

File: Assets/Editor/SectorGenerator.cs [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    {
+        SectorGenerator.GetWindow(typeof(SectorGenerator));
+    }
+
+    private const string ChunksFolder = "Chunks";
+    private const string MaterialsFolder = "Materials/Terrains";
+    private const string PrefabOutFolder = "Prefabs/Terrains";
+
+    private int _selected = 0;
+    string[] _options = null;
```
Comment (jaanonim) [inline Assets/Editor/SectorGenerator.cs:23]: missing private

File: Assets/Editor/SectorGenerator.cs [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+    private void OnGUI()
+    {
+        _options = GetSectorFolders();
+        EditorGUI.BeginChangeCheck();
+
+        this._selected = EditorGUILayout.Popup("My Simple Dropdown", _selected, _options);
```
Comment (jaanonim) [inline Assets/Editor/SectorGenerator.cs:30]: Love name of the popup

File: Assets/Editor/SectorGenerator.cs [hunk 3]
```diff
@@ -0,0 +1,119 @@
+using System;
+using UnityEngine;
+using UnityEditor;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Unity.VisualScripting;
```
Comment (jaanonim) [inline Assets/Editor/SectorGenerator.cs:7]: what you did with VisualScripting?

File: Assets/Editor/SectorGenerator.cs [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+{
+    [MenuItem("Window/SectorGenerator")]
+    public static void ShowWindow()
+    {
+        SectorGenerator.GetWindow(typeof(SectorGenerator));
+    }
+
+    private const string ChunksFolder = "Chunks";
+    private const string MaterialsFolder = "Materials/Terrains";
+    private const string PrefabOutFolder = "Prefabs/Terrains";
```
Comment (jaanonim) [inline Assets/Editor/SectorGenerator.cs:20]: Maybe add those path as window input fields so they can be esasly cheange. (not nessecery)

**Review by jaanonim — CHANGES_REQUESTED** [review]

**Review by Mimilo13 — APPROVED** [review]

**Pull request closed without merging**